packages/bls12-381-gen/bls12-381-gen.0.4.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Functors to generate BLS12-381 primitives based on stubs"
+description: "Functors to generate BLS12-381 primitives based on stubs"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "ocaml" {>= "4.08" & < "4.12"}
```

Comment:
One last question and it should be good to go: why is it not available on 4.12? If there is any issue I'll fix your package but if that's "just in case" I'd rather not to have that. That would add work to both of us:
- you by making another release for 4.12
- me by having to understand why the package was constrained in the first place and pushing the change anyway.

If it's a real incompatibility however that's totally fine.

Reply:
It is compatible with 4.12, but the CI here was failing because jsoo was not available for 4.12. I thought adding the dep was the way to go.

Reply:
I'm surprised, was it really failing or just skipped? It's not supposed to fail.